TENDemo/AgoraManager/AgoraManager.swift

```diff
@@ -40,6 +40,8 @@ open class AgoraManager: NSObject, ObservableObject {
     @Published var label: String?
     @Published var sessionStatus = SessionStatus.loading
 
+    let sttStreamDecoder = STTStreamDecoder()
```

Comment:
While the name of the variable is pretty straight forward, it's key to decoding so we don't want the reader to mis this. This line should include a comment to explain that this is the decoder provided by Agora.